api/app/Actions/EventType/GetEventTypeByIdAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\EventType;
+
+use App\Actions\GetByIdRequest;
+use App\Repositories\EventType\EventTypeRepository;
+
+final class GetEventTypeByIdAction
+{
+    private $repository;
+
+    public function __construct(EventTypeRepository $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function execute(GetByIdRequest $request): GetEventTypeByIdResponse
+    {
+        $eventType = $this->repository->getById($request->getId());
```

Comment:
After applying changes to the repository, throw ModelNotFound if $eventType is null.